packages/flame/lib/src/text.dart

```diff
@@ -93,125 +100,23 @@ abstract class BaseTextConfig {
   });
 }
 
-/// An extension of the BaseTextConfig which includes more configs supported by
-/// TextPaint
-class TextPaintConfig extends BaseTextConfig {
-  /// The font color to be used.
-  ///
-  /// Dart's [Color] class is just a plain wrapper on top of ARGB color (0xAARRGGBB).
-  /// For example,
-  ///
-  ///     const TextPaint config = TextPaint(color: const Color(0xFF00FF00)); // green
-  ///
-  /// You can also use your Palette class to access colors used in your game.
-  final Color color;
-
-  /// The font family to be used. You can use available by default fonts for your platform (like Arial), or you can add custom fonts.
-  ///
-  /// To add custom fonts, add the following code to your pubspec.yaml file:
-  ///
-  ///     flutter:
-  ///       fonts:
-  ///         - family: 5x5
-  ///           fonts:
-  ///             - asset: assets/fonts/5x5_pixel.ttf
-  ///
-  /// In this example we are adding a font family that's being named '5x5' provided in the specified ttf file.
-  /// You must provide the full path of the ttf file (from root); you should put it into your assets folder, and preferably inside a fonts folder.
-  /// You don't need to add this together with the other assets on the flutter/assets bit.
-  /// The name you choose for the font family can be any name (it's not inside the TTF file and the filename doesn't need to match).
-  final String fontFamily;
-
-  /// Creates a constant [TextPaint] with sensible defaults.
-  ///
-  /// Every parameter can be specified.
-  const TextPaintConfig({
-    this.color = const Color(0xFF000000),
-    this.fontFamily = 'Arial',
-    double fontSize = 24.0,
-    TextDirection textDirection = TextDirection.ltr,
-    double? lineHeight,
-  }) : super(
-          fontSize: fontSize,
+/// It does not hold information regarding the position of the text to be
+/// rendered, nor does it contain the text itself (the string).
+/// To use that information, use the [TextComponent], which uses [TextPaint].
+class TextPaint extends TextRenderer {
+  final MemoryCache<String, TextPainter> _textPainterCache = MemoryCache();
+
+  TextPaint({TextStyle? style, TextDirection? textDirection})
+      : super(
+          style: style ??
+              const TextStyle(
```

Comment:
Can we extract this to a variable?

Reply:
Tried before, but forgot to put `static` on it. Now it worked. :)